packages/optimizely-sdk/lib/shared_types.ts

```diff
@@ -0,0 +1,14 @@
+export type UserAttributes = {
+   // TODO[OASIS-6649]: Don't use any type
+   // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+   [name: string]: any;
+}
+
+export type Condition = {
+   name: string;
+   type: string;
+   match: string;
```

Comment:
The evaluator has the ability to handle when `match` is missing (in that case it is treated as an exact match), so let's capture that here:
```suggestion
   match?: string;
```